docs/docker.rst

```diff
@@ -113,3 +113,11 @@ executable in a bare-metal installation: ::
         /data /out/fmriprep-<latest-version> \   | These lines
         participant \                            | correspond to
         -w /work                                 | fmriprep arguments.
+
+.. warning::
+
+   When using docker with big datasets (+10GB) docker might fail. 
+   Changing the maximum size of the container will solve it: ::
+
+    $ service docker stop
+    $ dockerd --storage-opt dm.basesize=30G
```

Comment:
You should specify what OS/service managers this works with. I'm guessing it's systemd?

Reply:
If this is an issue of the working directory not being mounted, shall we just discard it? Or change the warning to "make sure you mounted the working dir"?

Reply:
t might be worth mentioning something about the memory allocated here ? Could optionally link out:
https://docs.docker.com/config/containers/resource_constraints/

Reply:
(if we're adding warnings, but +1 on clarifying re working directory being mounted !)